netparams/defaults.go

```diff
@@ -29,6 +29,7 @@ const (
 
 func defaultNetParams() map[string]value {
 	return map[string]value{
+		ProtocolUpgradeProposalRequiredMajority: NewDecimal(DecimalGTE(num.MustDecimalFromString("0.5")), DecimalLTE(num.MustDecimalFromString("1"))).Mutable(true).MustUpdate("0.66"),
```

Comment:
Why not just reuse the `ValidatorsVoteRequired` ?

Reply:
we probably can, but not sure we have to bind it to protocol implementation network param. Anyways, can change to us it and spare one network parameter